mlflow/store/artifact/models_artifact_repo.py

```diff
@@ -59,12 +59,11 @@ def split_models_uri(uri):
     def get_underlying_uri(uri):
         # Note: to support a registry URI that is different from the tracking URI here,
         # we'll need to add setting of registry URIs via environment variables.
-        from mlflow.tracking import MlflowClient
```

Comment:
For local imports, let's not try to import from the module level. We can change the reference here to `from mlflow import MlflowClient` but the local import is desired here.